resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -191,6 +191,8 @@ class KubernetesSuite extends SparkFunSuite
       .set("spark.kubernetes.driver.pod.name", driverPodName)
       .set("spark.kubernetes.driver.label.spark-app-locator", appLocator)
       .set("spark.kubernetes.executor.label.spark-app-locator", appLocator)
+      .set("spark.kubernetes.executor.request.cores", "0.5")
+      .set("spark.kubernetes.driver.request.cores", "0.5")
```

Comment:
Although this is only for tests, I'd like not to recommend this way. The driver has many threads and the flakiness is increasing when the driver CPU are capped.